### ISSUE TITLE
Adding triggering parameters for HA/ZHA

### ISSUE THREAD

**Issue by MattWestb** [thegroove/esphome-zeroconf#1]
I have not reading how its working but have seen tube0013 was added in HA and is triggering on some name parameters getting from the device / network.

I think it can being god putting in one "working name" for the serial ZBGW so HA can finding it and knowing how to using it.

Perhaps "ESPHomeZBGW-XXXX" then XXXX is the last part of the device MAC or some thing that is unique and also always being identified by HA and with the last part for adding more then one devices (or HA is doing that part in its side).

Perhaps also adding coordinator type EZSP/TI/Zigate and so on and baud rate and flow control to HA so it can using it hen adding ZHA.

If all is going well it can being possible triggering installation of ZHA with only clicking OK from the user and all is sett and done.

**Comment by thegroove** [comment]
Yes, this component is a step towards that, as we can now advertise in any way we want. For ZHA to identify a specific instance of a gateway, I think it makes the most sense considering how zeroconf aims to work, to have HA/ZHA to store the <whatevertheusersets>.local name. It's technically always unique on a network, and is query-able over mDNS to get an IP-address. This is also how the ESPHome integration in Home Assistant works. The service name can reflect the protocol, 'ezsp', etc., and additional things like baud_rate, etc., would go into the txt block.

**Comment by Hedda** [comment]
FYI, [Tube's Zigbee Gateways](https://github.com/tube0013/tube_gateways) are now (only) device listed in ZHA docs as discoverable via Zeroconf, see:

[https://www.home-assistant.io/integrations/zha/#discovery-via-usb-or-zeroconf](https://www.home-assistant.io/integrations/zha/#discovery-via-usb-or-zeroconf)

More discussion here:

https://community.home-assistant.io/t/zha-automatic-discovery-of-zigbee-coordinator-bridges-gateways-ethernet-wifi-network-devices-that-support-zeroconf-or-ssdp/293300/
